Docs/dashboard dynamic statistics

lucaball opened the pull request:
With this PR I provide documentation for creating custom statistics. With was introduced with #232

lucaball committed:
Init writing docs for Dashboard items

lucaball committed:
Documentation finetuning.
Add basic example and some grammar updates

lucasballyn commented:
Hi @loic425, 

I was not sure if I also had to push the built documentation or not. If so, please let me now. I'll push it.

lucaball committed:
Some small enhancments because of travis ci failure

loic425 commented on docs/cookbook/dashboard/index.rst:
```diff
@@ -0,0 +1,39 @@
+---------------------
+
+Since Symfony 4.4 it is possible to sort your services with a static function called ``getDefaultPriority``.
+Here you need to return an integer to set the weight of the service. Statistics with a higher priority will be displayed first.
```
Yes and I think we can add:
This is why we choose to use a negative value (**-1** for the first element, **-2** for the second...).

loic425 commented on docs/cookbook/dashboard/index.rst:
```diff
@@ -0,0 +1,39 @@
+    }
+
+
+.. note::
```
maybe a warning instead? What do you think?

loic425 commented on docs/cookbook/dashboard/index.rst:
```diff
@@ -0,0 +1,39 @@
+
+Create your own statistic
+-------------------------
+Add a new class to ``/src/Dashboad/Statistic`` and make sure it implement the ``App\Dashboard\Statistics\StatisticInterface``.
```
``/src/Dashboard/Statistic``
There's a typo on "Dashboard"

lucaball committed:
Fix typo

lucaball committed:
Add explanation for choise of sequence.
Also encourage users to use their own sequence if they want to

lucaball committed:
Change note to clear cache to warning
So users won't miss it more easily. Because a warning draws the attention faster.

loic425 commented:
Thanks @lucasballyn :1st_place_medal:
